packages/fast-form-generator-react/src/form/form-item.align-horizontal.tsx

```diff
@@ -35,13 +44,27 @@ class FormItemAlignHorizontal extends React.Component<IFormItemComponentMappingT
             });
 
             if (typeof option !== "undefined") {
+                let className: string = "";
+
+                switch (direction) {
+                    case "left":
+                        className = this.props.managedClasses.formItemAlignHorizontal_input__left;
+                        break;
+                    case "center":
+                        className = this.props.managedClasses.formItemAlignHorizontal_input__center;
+                        break;
+                    case "right":
+                        className = this.props.managedClasses.formItemAlignHorizontal_input__right;
+                }
```

Comment:
you can put this as a private method to like getInputClassName and just do `const className: string = this.getInputClassName(direction);`

Reply:
Fixed.